src/data.upgrade.func.php

```diff
@@ -106,7 +110,12 @@ function litespeed_update_5_3()
 {
 	global $wpdb;
 	Debug2::debug('[Data] Upgrade url_file table');
-	$tb_exists = $wpdb->get_var('SHOW TABLES LIKE "' . $wpdb->prefix . 'litespeed_url_file"');
+
+	$save_state = $wpdb->suppress_errors;
```

Comment:
@timotei-litespeed this duplicate lines could be a func. I will update it. Just remind you.